### DISCUSSION TITLE
HomeAssistant SkyConnect any plans to provide support?

### DISCUSSION THREAD

**Discussion by leewilko** [Koenkk/zigbee2mqtt#19025] [Devices]
Hi,

I really want to move over to the SkyConnect dongle for Zigbee and Matter support but compatibility with Zigbee2MQTT (which i love) is stopping me from doing this.

Are there any firm plans to support this Co-ordinator device?

Thanks

Lee

**Comment by francisp2** [comment]
This works, both with the Zigbee stack, or with the multiprotocol stack on the SkyConnect.
use
```
serial:
  adapter: ezsp
```

**Reply by leewilko** [reply]
Is this now formally supported and stable as the website still states as Experimental, are there plans for the Skyconnect to move to recommended or is there elements still preventing this, I am holding off from making a switch to the Multiprotocol support for Matter on Z2MQTT for now and really dont want to switch to ZHA if I can avoid it?, any guidance or estimate/loose timeframes for improvement on Experimental would be amazing to understand?

Experimental
The adapters below are experimental, don't use these if you want a stable setup.

Based on Silicon Labs EFR32MG2x/MGM21x and EFR32MG1x/MGM1x series
Initial development started on experimental (beta stage) support for Silicon Labs based Zigbee adapters. This includes all EM35X, EFR32MG12, EFR32MG13, EFR32MG21, EFR32MG24 SoCs/Modules families with Silabs EmberZNet NCP 6.7.8 firmware or later via EZSP version 8 (EmberZNet Serial Protocol) interface.

USB connected

ITead Sonoff Zigbee 3.0 USB Dongle Plus V2 model "ZBDongle-E"
Home Assistant SkyConnect (by Nabu Casa)
Elelabs ELU013 and Popp ZB-STICK
CoolKit ZB-GW04 USB dongle (a.k.a. easyiot stick)
Z-Wave.Me Z-Station dual Zigbee & Z-Wave
SMLIGHT SLZB-07 Zigbee USB adapter with +3dB antenna

**Reply by cinghialino** [reply]
sorry, can you expand on this? do you mean to make Z2M work all you need to do is to change from 

`port: /dev/ttyUSB0` 

to 

```
serial:
  adapter: ezsp
```

**Reply by francisp2** [reply]
```
serial:
  port: /dev/ttyUSB0
  adapter: ezsp
```
